src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,200 @@
+        showToUser(LINE, message, LINE);
+    }
+
+    /**
+     * Method that warns user that there is a missing description for the task.
+     * @param command
+     */
+    public static void warningMissingDescription(String command) {
+        switch (command) {
+        case "todo":
+            String todoMessage = "OOPS!!! The description of a todo cannot be empty.";
+            showToUser(LINE, todoMessage, LINE);
+            break;
+        case "deadline":
+            String deadlineMessage = "OOPS!!! The description of a deadline cannot be empty.";
+            showToUser(LINE, deadlineMessage, LINE);
+            break;
+        case "event":
+            String eventMessage = "OOPS!!! The description of an event cannot be empty.";
+            showToUser(LINE, eventMessage, LINE);
+            break;
+        }
+    }
+
+    /**
+     * Method to show goodbye message when user exits programme with command bye
+     */
+    public static void showGoodbyeMessage() {
+        String message = "     Bye. Hope to see you again soon!";
+        showToUser(LINE, message, LINE);
+    }
+
+    /**
+     * Method to show message prompting user that command is invalid or wrong
+     */
+    public static void showInvalidInputMessage() {
+        String message = "OOPS!!! I'm sorry, but I don't know what that means :-(";
+        showToUser(LINE, message, LINE);
+    }
+
+    /**
+     * Method to show message after successfully marking task as complete
+     */
+    public static void showCompletedMessage() {
+        String message = "Nice! I've marked this task as done:";
+        showToUser(LINE, message);
+    }
+
+    /**
+     * Method to show line divider
+     */
+    public static void showLine() {
+        System.out.println(LINE);
+    }
+
+    /**
+     * Method to show added task, along with uodated number of tasks in list
+     * @param taskName
+     * @param numTasks
+     */
+    public static void showAddedTask(String taskName, int numTasks) {
+        String successfulMessage = "Got it. I've added this task:";
+        String updatedListNumber = "Now you have " + numTasks + " task(s) in the list.";
+        showToUser(LINE, successfulMessage, taskName, updatedListNumber, LINE);
+    }
+
+    /**
+     * Shows list of all the tasks that have not been deleted
+     */
+    public static void showList() {
+        String message = "Here are the tasks in your list:";
+        showToUser(LINE, message);
+    }
+
+    /**
+     * Shows message if number does not correspond to any task
+     */
+    public static void noTask() {
+        String message = "No such task exists.";
+        showToUser(LINE, message, LINE);
+    }
+
+    /**
+     * Show message upon successfully deleting a task
+     */
+    public static void showSuccessfulDelete() {
+        String successfulMessage = "Noted I've removed this task:";
+        showToUser(LINE, successfulMessage);
+    }
+
+    /**
```

Comment:
There should be a space between the description and the @param in header comments. I noticed this issues a couple other times.